README.md

```diff
@@ -69,6 +69,8 @@ The table below lists the dependencies of the Data SDK.
 | Snappy               | 1.1.7           |
 | RapidJSON            | latest          |
 
+### Additional Linux dependencies
```

Comment:
```suggestion
### Linux dependencies
```

Reply:
Changed locally.
I will commit the change with all other implementations.